app/src/js/AstroMath.js

```diff
@@ -21,26 +24,19 @@ export default class AstroMath {
    * dMajorRadius and dMinorRadius that corresponds to that specific target.
    * @param  { String } targetName - the name of the specific target.
    */
-  constructor(targetName) {
-    this.targetName = targetName;
+    constructor(targetName) {
+      this.targetName = targetName;
+
+      var targets = MY_JSON_MAPS['targets'];
+      for(var i = 0; i < targets.length; i++) {
```

Comment:
I think we want to use let instead of var for ES6.

Reply:
fixed